pom.xml

```diff
@@ -13,8 +13,7 @@
     <url>http://wiki.jenkins-ci.org/display/JENKINS/Mercurial+Plugin</url>
     
     <properties>
-      <jenkins.version>1.580.1</jenkins.version>
-      <java.level>6</java.level>
+      <jenkins.version>1.642.3</jenkins.version>
```

Comment:
Need a new baseline to be able to run tests against reasonably modern versions of Pipeline; the 1.4.x line compatible with 1.580.x did not even include `workflow-multibranch`, which most naturally reproduces this error.